src/components/Dashboard/Organizations/index.jsx

```diff
@@ -1,15 +1,89 @@
-import {Box, Button, ButtonGroup, Divider, Typography} from "@mui/material";
-import { useState } from "react";
+import {Box, Button, ButtonGroup, Divider, Typography, CircularProgress, Link} from "@mui/material";
+import { useState, useEffect } from "react";
 import { vars } from "../../../theme/variables";
 import { ListIcon, TableChartIcon } from "../../../Icons";
 import AddOutlinedIcon from "@mui/icons-material/AddOutlined";
-import organizationss from "../../../static/Organizations.json";
 import OrganizationsList from "../../common/OrganizationsList";
 import CustomViewButton from "../../common/CustomViewButton";
+import { createNewOrganization, getOrganizations } from "../../../api/endpoints/apiService";
+import BasicDialog from "../../common/BasicDialog";
+
+const { gray600, gray200, brand700, brand800 } = vars;
+
+const linkStyles = {
+  color: brand700,
+  fontWeight: 600, 
+  textDecoration: "none", 
+  "&:hover": { 
+    color: brand800 
+  }
+}
 
-const { gray600, gray200 } = vars;
 const Organizations = () => {
   const [listView, setListView] = useState('list');
+  const [organizations, setOrganizations] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [message, setMessage] = useState();
+  const [open, setOpen] = useState(false);
+
+  // TODO: change this to be dynamic when we get the response from api call
+  const groupname = "aigul"
+
+  const handleOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const fetchOrganizations = async() => {
```

Comment:
@Aiga115 this code is exactly the same we have in src/components/organizations/index.jsx, could you factor this out in a util module so that we can re-use the same code here and in the other file? If you need to you can pass the hooks as parameter, but at least we avoid to repeat code that needs to be maintained. Thanks